native-schema-registry/python/PyGsrSerDe/AvroSerDe.py

```diff
@@ -0,0 +1,42 @@
+import fastavro
+from io import BytesIO
+
+
+class AvroSerializer:
+
+    def parse_schema(self, schema):
```

Comment:
Can you please add doc string for class and functions?
For reference:
https://www.programiz.com/python-programming/docstrings

I will also suggest you to setup a python linter/formatter such that it will automatically generate these docstrings for methods and classes and warns you when they are missing

Reply:
In addition, you should use type-hinting like serialize method below

Reply:
Added.